### PR TITLE
New Map filter design 

### PR DESCRIPTION
**Pull request by l2succes**
#skip_new_tests

### CODE REVIEW

File: Pod/Classes/ViewControllers/ARComponentViewController.h [hunk 1]
```diff
@@ -5,10 +5,19 @@
...
 NS_ASSUME_NONNULL_BEGIN
 
-@interface ARComponentViewController : UIViewController
+@protocol ARComponentFullBleedViewSizing
+
+@property (nonatomic, assign) BOOL fullBleed;
+
+@end
+
+@interface ARComponentViewController : UIViewController <ARComponentFullBleedViewSizing>
```
Comment (orta) [inline Pod/Classes/ViewControllers/ARComponentViewController.h]: I'm not sure that this protocol adds much value - as every component VC supports it, the Eigen subclass can use the ARMenuAware protocol on that side of the fence though

File: Pod/Classes/ViewControllers/ARMapComponentViewController.m [hunk 2]
```diff
@@ -7,6 +7,21 @@ - (instancetype)init
...
+                                [NSLayoutConstraint constraintWithItem:self.rootView
+                                                             attribute:NSLayoutAttributeTop
+                                                             relatedBy:NSLayoutRelationEqual
+                                                                toItem:self.view
+                                                             attribute:NSLayoutAttributeTop
+                                                            multiplier:1
+                                                              constant:0]
+                                ]];
+}
+
```
[the remark this answers is not in the document]
Comment (l2succes) [reply Pod/Classes/ViewControllers/ARMapComponentViewController.m]: No, that's not needed anymore

File: Pod/Classes/ViewControllers/ARComponentViewController.h (hunk 1 above)
Comment (orta) [reply Pod/Classes/ViewControllers/ARComponentViewController.h]: This is also used on a non-ARComponentVC, so there's value

**Comment by orta** [comment]
Alright, this has been rebased and brought up to master post https://github.com/artsy/emission/pull/1345

**Comment by orta** [comment]
OK, looking good. Merging 👍

**Pull request merged**